products/protractor.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Protractor
+category: framework
+tags: google javascript-runtime test
+iconSlug: protractor
+permalink: /protractor
+versionCommand: npm list protractor
+changelogTemplate: https://github.com/angular/protractor/blob/master/CHANGELOG.md
+releaseDateColumn: true
+activeSupportColumn: Community Support
+extendedSupportColumn: Commercial Support
+eolColumn: false
+
+auto:
+-   npm: protractor
+
+identifiers:
+-   purl: pkg:npm/protractor
+
+releases:
+-   releaseCycle: "7.0"
+    support: 2023-08-31
+    eol: true
+    extendedSupport: true
+    latest: "7.0.0"
+    latestReleaseDate: 2020-05-13
+    releaseDate: 2020-05-13
+    link: https://github.com/angular/protractor/commit/71771dee4e87526dffb725cacac3f9f9e1efc287?short_path=06572a9#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed
+
+-   releaseCycle: "6.0"
+    support: 2023-08-31
+    eol: true
+    extendedSupport: false
+    latest: "6.0.0"
+    latestReleaseDate: 2190-03-23
+    releaseDate: 2019-03-23
+    link: https://github.com/angular/protractor/commit/5d8da049815ffc104e6eca71d2ecaec51b00f520?short_path=06572a9#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed
+
+-   releaseCycle: "5.0"
+    support: 2023-08-31
+    eol: true
+    extendedSupport: false
+    latest: "5.4.4"
+    latestReleaseDate: 2020-04-16
+    releaseDate: 2017-01-09
+    link: https://github.com/angular/protractor/commit/a0ffa9b900cf8aa7123c634341e2d9fa9c498db1?short_path=06572a9#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed
+
+-   releaseCycle: "4.0"
+    support: 2023-08-31
+    eol: true
+    extendedSupport: false
+    latest: "4.0.13"
```

Comment:
issue: According to the [changelog](https://github.com/angular/protractor/blob/master/CHANGELOG.md#4014) the last version of 4.0 was 4.0.14

Reply:
good catch done